### DISCUSSION TITLE
Invalid credentials error not being caught?

### DISCUSSION THREAD

**Discussion by bquek00** [supabase/supabase#17818] [Questions]
I am using a try catch but whenever I give it invalid credentials, the 400 bad request error is still in my console in red. Is there anyway to remove it? Thank you.

try {
          const { error, data } = await supabase.auth.signInWithPassword({email: email, password: password });
          if (error) {
            console.log('Login error:', error.message);
            alert('Invalid Credentials');
          } else {
            await new Promise(resolve => router.push('/auth/callback', undefined, { shallow: true }, resolve));
            }
        } catch (error) {
          console.error('Login error:', error.message);

**Comment by GaryAustin1** [comment]
In general supabase-js passes the errors back in error and do not generate a catch-able error condition.
